bindings/nodejs/src/lib.rs

```diff
@@ -23,3 +36,100 @@ pub fn debug() -> String {
         .finish();
     format!("{:?}", op.metadata())
 }
+
+
+#[napi]
+pub struct OperatorFactory {}
+
+#[napi]
+impl OperatorFactory {
+    #[napi]
+    pub fn memory() -> napi::Result<OpendalStore> {
+        let op = Operator::create(services::Memory::default())
+            .unwrap()
+            .finish();
+
+        Ok(OpendalStore::new(op))
+    }
+}
+
+#[allow(dead_code)]
+#[napi]
+pub struct ObjectMeta {
+    pub location: String,
+    pub last_modified: i64,
+    pub size: u32
+}
+
+#[napi]
+pub struct OpendalStore {
```

Comment:
**suggestion**: Can we keep the same API unless we have limitations? Use `pub struct Operator(opendal::Operator)` instead.

Reply:
SGTM.